doc/man1/enc.pod

```diff
@@ -199,12 +199,12 @@ encrypting a file and read from the encrypted file when it is decrypted.
 
 Some of the ciphers do not have large keys and others have security
 implications if not used correctly. A beginner is advised to just use
-a strong block cipher in CBC mode such as bf or des3.
+a strong block cipher in CTR mode such as AES or Camellia.
```

Comment:
> We're dealing with beginners here

I'm not convinced that advising beginner to use CTR for *file* encryption is a good idea. In fact I'm not at all convinced that CTR is suitable mode for *file* encryption even in more general sense. By extension I'm not convinced that GCM should be supported in enc command, i.e. ever. I mean I can appreciate the idea of enc command providing authenticated mode, but even then I'm not convinced that it should support GCM. I'd argue that right mode to recommend first for file encryption is one *designed* for storage, e.g. XTS. But in such case one has to remember that it has cryptographically-significant parameter, block size. So that formally speaking enc command should either stick to fixed block size or save initial one in a header. But this is going beyond scope of this request. In the context of this request I'd also vote against implying that CTR is preferred mode for enc command.